recipes/sdf/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "sdf" %}
+{% set version = "0.3.3" %}
+{% set sha256 = "c1c7dfe08d0294dfa179c50f8f6b8a04d1a0195c3b50bf259a73dac37da75a40" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/SDF-{{ version }}.tar.gz  
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```

Comment:
Could this be a noarch package?

Reply:
With 'noarch: python' set the build complains about the native libraries contained in the package. The Python code itself does not depend on the Python version or platform but one module uses native code (loaded with ctypes).

Reply:
Ah oh well

Reply:
feel free to revert (I'll merge once you do)

Reply:
I didn't realize that you had C code, do you compile it during the build process?

Reply:
The GitHub repository (https://github.com/ScientificDataFormat/SDF-Python) contains pre-built versions of the shared libs for all supported platforms, so the universal wheel can be built on each platform. Build scripts for all supported platforms are also contained.